docs/docs/getting-started.md

```diff
@@ -4,82 +4,82 @@ title: Getting Started
 
-The easiest way to start hacking on React is using the following JSFiddle Hello World examples:
+## Trying Out React
 
- * **[React JSFiddle](https://jsfiddle.net/reactjs/69z2wepo/)**
- * [React JSFiddle without JSX](https://jsfiddle.net/reactjs/5vjqabv3/)
+If you're just interested in playing around with React, you can use JSFiddle. Try starting from [this Hello World example code](https://jsfiddle.net/o9gspf3e/). You don't need to install anything; you can just modify the code and click "Run" to see if it works.
 
-## Create React App
+## Creating a New Application
 
-**[Create React App](http://github.com/facebookincubator/create-react-app)** is a new officially supported way to create single-page React applications. It offers a modern build setup with no configuration. It requires Node 4 or higher.
+[`create-react-app`](http://github.com/facebookincubator/create-react-app) is the best way to starting building a new React single page application. It handles webpack, babel, and live reloading, so you don't have to think about them.
 
-Note that it has [some limitations](https://github.com/facebookincubator/create-react-app#limitations) and is only useful for single-page applications. If you need more flexibility, or if you want to integrate React into an existing project, consider other options below.
+```bash
+npm install -g create-react-app
+create-react-app hello-world
+cd hello-world
+npm start
+```
+
+`create-react-app` doesn't handle backend logic or databases; it just creates a frontend build pipeline, so you can use it with any backend you want.
 
-## Starter Pack
+## Adding React to an Existing Application
 
-If you're just getting started, you can download the starter kit. The starter kit includes prebuilt copies of React and React DOM for the browser, as well as a collection of usage examples to help you get started.
+We recommend using React from npm with a bundler like [browserify](http://browserify.org/) or [webpack](https://webpack.github.io/). If you already have npm in your build pipeline, you can install it with:
 
-<div class="buttons-unit downloads">
-  <a href="/react/downloads/react-{{site.react_version}}.zip" class="button">
-    Download Starter Kit {{site.react_version}}
-  </a>
-</div>
+```bash
+npm install --save react react-dom
+```
 
-In the root directory of the starter kit, create a `helloworld.html` with the following contents.
+and import it from your code with:
+
+```js
+var React = require('react');
+var ReactDOM = require('react-dom');
+ReactDOM.render(<App />, document.getElementById('the-element-id'));
+```
+
+If you use Bower, React is available via the `react` package:
+
+```bash
+bower install --save react
+```
+
+## Using React From A CDN
+
+If you don't have any build pipeline set up, you can still use React directly from a CDN. You also need to load `babel` from a CDN and load your scripts with type `text/babel` so that they can get transformed. This code works as a single html file:
```

Comment:
I would completely remove this section. It conflates two different things:

* Using React from CDN (fine for dev and prod)
* Using in-browser Babel transform (exists only for trying React and literally nothing else)

This section gives a wrong impression that somehow using React from CDN is bad, which it is not.

I think, as I mentioned above, the section on React CDN files should be merged with `Adding React to an Existing Application`, and the part about in-browser Babel can be fully omitted and instead added to a downloadable `index.html` in `Trying Out React`.